src/accelerate/state.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         self.__dict__ = self._shared_state
         if parse_flag_from_env("USE_CPU"):
             cpu = True
+        self.use_fork = parse_flag_from_env("USE_LAUNCHER", 0)
```

Comment:
```suggestion
        self.fork_launched = parse_flag_from_env("FORK_LAUNCHED", 0)
```
Might be clearer this way.

Reply:
Makes sense!